_sources/SequenceMutation/Mutability.rst

```diff
@@ -0,0 +1,184 @@
+..  Copyright (C)  Brad Miller, David Ranum, Jeffrey Elkner, Peter Wentworth, Allen B. Downey, Chris
+    Meyers, and Dario Mitchell.  Permission is granted to copy, distribute
+    and/or modify this document under the terms of the GNU Free Documentation
+    License, Version 1.3 or any later version published by the Free Software
+    Foundation; with Invariant Sections being Forward, Prefaces, and
+    Contributor List, no Front-Cover Texts, and no Back-Cover Texts.  A copy of
+    the license is included in the section entitled "GNU Free Documentation
+    License".
+
+.. qnum::
+   :prefix: seqmut-1-
+   :start: 1
+
+.. index:: mutable, immutable
+   runtime error; string item assignment
+   string; immutable
+
+Mutatability
+============
+
+Some Python collection types - strings and lists so far - are able to mutate and some are not. If 
```

Comment:
Need to define both mutate and mutable.  Or we could just substitute the word change for mutate in this opening sentence.